web-client/src/presenter/sequences/JudgeActivityReport/submitJudgeActivityReportSequence.ts

```diff
@@ -27,8 +27,8 @@ export const submitJudgeActivityReportSequence = showProgressSequenceDecorator([
   validateJudgeActivityReportSearchAction,
   {
     error: [
-      setAlertErrorAction,
```

Comment:
Throughout, I have removed `setAlertErrorAction` when it precedes `setValidationAlertErrorsAction` since the latter overwrite the former.